tsup.config.ts

```diff
@@ -1,18 +1,16 @@
 import type { Options } from './tsup';
 
+
 import { defineConfig } from 'tsup';
-import { DIR_DST, DIR_DST_ASSETS, DIR_DST_STATIC } from './tsup/constants';
+import { DIR_DST, DIR_DST_ASSETS } from './tsup/constants';
 
 
-export default defineConfig((options: Options) => {
-	if (options.d === DIR_DST) {
-		return import('./tsup/server').then(m => m.default());
-	}
-	if (options.d === DIR_DST_ASSETS) {
-		return import('./tsup/client-asset').then(m => m.default());
-	}
-	if (options.d === DIR_DST_STATIC) {
-		return import('./tsup/client-static').then(m => m.default());
-	}
-	throw new Error(`Unconfigured directory:${options.d}!`)
-})
+export default defineConfig(async (options: Options) => {
+  if (options.d === DIR_DST) {
+    return import('./tsup/server').then(m => m.default());
+  }
+  if (options.d === DIR_DST_ASSETS) {
+    return import('./tsup/client-asset').then(m => m.default());
```

Comment:
```suggestion
    return import('./tsup/client').then(m => m.default());
```

Reply:
Since we're not going to have static for now it would be less confusing to call it simply "`client`".

Reply:
That is fine, but that comes from starter-ts